wordpress-mobile/WordPress-iOS
# Blaze: Add entry point for menu item

Part of #20091 

- Adds an entry point for the dashboard menu item
- Button actions will be addressed in a different PR
- Analytics will be addressed in a differnt PR
- WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/577
- Figma: cEGtlsm6xijEswaKkrQrIj-fi-477%3A5675

## How to test
1. Switch to a site that's eligible for Blaze (See peeHDf-zm-p2 for more info on Blaze requirements)
2. ✅ A blaze menu item is displayed
3. Switch to a site that's not eligible for Blaze
4. ✅ A blaze menu item isn't displayed

Blaze available | Blaze unavailable
-- | --
![Simulator Screen Shot - iPhone 14 Pro - 2023-02-12 at 18 35 36](https://user-images.githubusercontent.com/6711616/218330409-f5c2daca-1e36-4494-8a34-8b52459a9a51.png) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-02-12 at 18 35 42](https://user-images.githubusercontent.com/6711616/218330412-129b2730-28b7-4cf3-b1c3-aa3e7c574fee.png)


## Regression Notes
1. Potential unintended areas of impact
n/a

6. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

7. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
